dev_guide/daemonsets.adoc

```diff
@@ -19,8 +19,36 @@ A daemonset can be used to run replicas of a pod on specific or all nodes in an
 Use daemonsets to create shared storage, run a logging pod on every node in
 your cluster, or deploy a monitoring agent on every node.
 
+For security reasons and by their nature creating DaemonSets is restricted to cluster administrators.
+
```

Comment:
@tnozicka Do we need the _and by their nature_? 
For security reasons, only cluster administrators can create DaemonSets.

Reply:
I'll rephrase